packages/ButtonLink/ButtonLink.jsx

```diff
@@ -87,6 +100,7 @@ ButtonLink.propTypes = {
    * Target URL
    */
   href: PropTypes.string,
+  fullwidth: PropTypes.string,
```

Comment:
Please change this PropType to a boolean, and add comments above such as:

```jsx
/**
 * If `true`, sets `ButtonLink` to 100% width.
 * @since 2.2.0
 */
fullWidth: PropTypes.bool,
```